datahub-web-react/src/app/entity/container/preview/Preview.tsx

```diff
@@ -62,7 +63,7 @@ export const Preview = ({
     externalUrl?: string | null;
 }): JSX.Element => {
     const entityRegistry = useEntityRegistry();
-    const typeName = (subTypes?.typeNames?.length && subTypes?.typeNames[0]) || 'Container';
+    const typeName = capitalizeFirstLetterOnly(subTypes?.typeNames?.[0]) || 'Container';
```

Comment:
Minor: Will this work when 

subTypes?.typeNames is an empty array?. If not, seems a bit worrisome to remove the first check.